src/Modules/AuthModule/components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,6 +1,251 @@
+import { useForm } from "react-hook-form";
+import logo from "../../../../assets/images/PMS 3.svg";
+import { FormDataChangPass } from "../../../../interfaces/Auth";
+import "../Login/Login.css";
+import { useContext, useState } from "react";
+import { toast } from "react-toastify";
+import axios from "axios";
+import { AuthContext } from "../../../../context/AuthContext";
 
 export default function ChangePassword() {
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    watch,
+  } = useForm<FormDataChangPass>();
+
+  const { baseUrl } = useContext(AuthContext);
+
+  // Password eye
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+
+  const togglePasswordVisibility = (): void => {
+    setShowPassword((prevState: boolean) => !prevState);
+  };
+
+  const [showNewPassword, setShowNewPassword] = useState<boolean>(false);
+
+  const togglePasswordVisibility2 = (): void => {
+    setShowNewPassword((prevState: boolean) => !prevState);
+  };
+
+  const [showConfirmPassword, setShowConfirmPassword] = useState<boolean>(false);
+
+  const togglePasswordVisibility3 = (): void => {
+    setShowConfirmPassword((prevState: boolean) => !prevState);
+  };
+
+  const onSubmit = async (data: FormDataChangPass) => {
+    try {
+      let response = await axios.put(`${baseUrl}/Users/ChangePassword`,
+      data,
+      {
+        headers: { Authorization: `Bearer ${ localStorage.getItem("token")}` },
+      }
+    )
+      // logout();
+      toast.success('Password has been updated successfully');
```

Comment:
We should provide the message coming from the backend first and then default to the current message.